src/packagedcode/rpm_licenses.txt

```diff
@@ -1,2 +1,5 @@
 # TODO: populate me!
-public domain	public-domain
+public domain   public-domain
+lgplv2+   lgpl-2.0-plus
+gplv2+   gpl-2.0-plus
+lgplv2+ and gplv2+   lgpl-2.0-plus and gpl-2.0-plus  
```

Comment:
We would need a test for this. Also I am not sure that mapping an expression is the best way, as it could be better to map single license ids instead and use them with the license-expression library parser

Reply:
gentle ping. See https://github.com/nexB/scancode-toolkit/tree/develop/tests/licensedcode/data/datadriven/lic4 for actual examples for how to add a license test with many examples.